### PR TITLE
feat(St-Widget): Create Widget component

### PR DESCRIPTION
**Pull request by calcerrada**
CCT-135

### Documentation
- [x] Add the issue in PR description
- [x] Have changelog updated
- [x] You fill the milestone value
- [x] You add some label
- [x] Have example running in demo app
- [x]  Review id on demo is the same of egeo folder

### Code
- [ ] Pass Sass lint task
- [ ] Have qaTags

### Tests
- [x] Have at least 70% tests coverage

### CODE REVIEW

File: src/lib/st-widget/st-widget.component.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+ */
+
+@Component({
+   selector: 'st-widget',
+   templateUrl: 'st-widget.component.html',
+   styleUrls: ['st-widget.component.scss']
+})
+
+export class StWidgetComponent {
+   /** @Input {string} [title] title will de displayed in the widget header */
```
Comment (EvaMGC) [inline src/lib/st-widget/st-widget.component.ts]: Change ' will de' to ' will be '

File: src/lib/st-widget/st-widget.component.scss [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+            font-size: $egeo-font-size-large;
+            color: $action-secondary-04;
+            float: right;
+         }
+      }
+
+      &__title{
+         text-transform: capitalize;
+         //theme
+         color: $neutral-12;
```
Comment (ppenalver) [inline src/lib/st-widget/st-widget.component.scss]: This must be in theme

File: src/lib/st-widget/st-widget.component.scss [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+ * This software is licensed under the Apache License, Version 2.0.
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the terms of the License for more details.
+ *
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+@import './../../theme/constants/colors';
+@import './../../theme/constants/fonts';
```
Comment (ppenalver) [inline src/lib/st-widget/st-widget.component.scss]: remove this lines, this is for theme

File: src/lib/st-widget/st-widget.component.scss [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+            }
+
+             .icon-actions:last-of-type{
+               margin-left: -13px;
+            }
+
+         }
+         .setting-action {
+            cursor: pointer;
+            font-size: $egeo-font-size-large;
```
Comment (ppenalver) [inline src/lib/st-widget/st-widget.component.scss]: For theme

**Pull request merged**